### PR TITLE
Since version compliance-operator-0.1.35 and file-integrity-operator-0.1.16, the channel is release-0.1

### PR DESCRIPTION
**Pull request by ahardin-rh**
Updated per discussion with @xiaojiey 

Preview builds:
https://deploy-preview-33852--osdocs.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-installation.html#installing-compliance-operator-cli_compliance-operator-installation

https://deploy-preview-33852--osdocs.netlify.app/openshift-enterprise/latest/security/file_integrity_operator/file-integrity-operator-installation.html#installing-file-integrity-operator-using-cli_file-integrity-operator-installation

### CODE REVIEW

File: modules/file-integrity-operator-installing-cli.adoc [hunk 1]
```diff
@@ -70,7 +63,7 @@ metadata:
   name: file-integrity-operator
   namespace: openshift-file-integrity
 spec:
-  channel: "${OC_VERSION}"
+  channel: "release-0.1"
```
Comment (xiaojiey) [inline modules/file-integrity-operator-installing-cli.adoc:66]: I am not sure do we need to point out it only suitable when version is equal or higher than file-integrity-operator.v0.1.16.

**Comment by xiaojiey** [comment]
@ahardin-rh Please also hold it until the latest version is released.

File: modules/file-integrity-operator-installing-cli.adoc (hunk 1 above)
Comment (mrogers950) [reply modules/file-integrity-operator-installing-cli.adoc:66]: We want everyone to install the latest, so I think this is OK.

Comment (xiaojiey) [reply modules/file-integrity-operator-installing-cli.adoc:66]: Okay.

**Comment by xiaojiey** [comment]
/lgtm
You could merge to 4.8 first. But for 4.6 and 4.7 cherry-pick, please hold on until latest advisory released.
I will notify you by slack. Thanks.

**Comment by ahardin-rh** [comment]
@xiaojiey Okay, thank you! I will hold until further notice.

**Comment by xiaojiey** [comment]
@ahardin-rh Please help to merge https://github.com/openshift/openshift-docs/pull/33852 for 4.8/4.7/4.6. Our advisory is SHIPPED LIVE now. Thanks a lot!

**Review by JAORMX — APPROVED** [review]

**Review by kalexand-rh — COMMENTED** [review]
LGTM

**Review by adellape — APPROVED** [review]

**Pull request merged**